### ISSUE TITLE
Custom Ore Chunks Over-writed

### ISSUE THREAD

**Issue by Helimyne** [BeetoGuy/LevelUp2#28]
Put in ores from three mods, the custom color get deleted, but it does generates the chunks.

**Comment by BeetoGuy** [comment]
It does work on my end. Can you upload the config file?

**Comment by Helimyne** [comment]
before loading minecraft
`
# Configuration file

blacklist {
    # Crops that won't be affected by farming growth skill, uses internal block name. No sync to client required. [default: []]
    S:"Crops for farming" <

     >
}


debug {
    # Forces Level Up! to restore external json files to default
    B:"Reset json files"=false
}


digloot {
    # Chances that a common loot drop will appear [range: 0 ~ 100, default: 85]
    I:"Common Digging Loot Chance"=85

    # Chances that a rare loot drop will appear [range: 0 ~ 100, default: 1]
    I:"Rare Digging Loot Chance"=1

    # Chances that an uncommon loot drop will appear [range: 0 ~ 100, default: 15]
    I:"Uncommon Digging Loot Chance"=15
}


general {
    # Always drop ore chunks on ore harvest
    B:"Always drop ore chunks"=false

    # Use ore chunks for ore doubling
    B:"Break ores into chunks"=true

    # All ores can be doubled, even if they don't have a chunk.
    B:"Duplicate any ore"=true

    # Colors for the end ore item
    I:"End Ore colors" <
        094747
        343434
     >

    # XP credit level for Mining Specialization
    I:"End Ore experience" <
        4
        2
     >

    # End ores that double from mining efficiency
    S:"End Ores to double" <
        oreFluidEnder
        oreFluidCrudeOilSand
     >

    # Does the furnace eject doubled items?
    B:"Furnace ejects bonus items"=false

    # Colors for the nether ore item
    I:"Nether Ore colors" <
        15064789
        0C5ABE
        E66700
        FEE503
     >

    # XP credit level for Mining Specialization
    I:"Nether Ore experience" <
        2
        5
        4
        2
     >

    # Nether ores that double from mining efficiency
    S:"Nether Ores to double" <
        oreQuartz
        oreCobalt
        oreArdite
        oreFluidGlowstone
     >

    # Does the player lose all levels on death?
    B:"Reset class on death"=true

    # Colors for the surface ore item
    I:"Surface Ore colors" <
        3421236
        12360064
        16576075
        6155509
        1564002
        16711680
        1653940
        16739601
        9416910
        207234249
        CFEAF9
        8597D1
        C7C7CC
        CECEA3
        0588F4
        6EFFFF
        232323
        A40606
        FFF8F0
        435D39
     >

    # XP credit levels for Mining Specialization
    I:"Surface Ore experience" <
        1
        1
        2
        4
        4
        2
        2
        1
        1
        1
        1
        1
        2
        4
        3
        3
        1
        2
        2
        3
     >

    # Ores that double from mining efficiency
    S:"Surface Ores to double" <
        oreCoal
        oreIron
        oreGold
        oreDiamond
        oreEmerald
        oreRedstone
        oreLapis
        oreCopper
        oreTin
        oreSilver
        oreLead
        oreAluminum
        oreNickel
        orePlatinum
        oreIridium
        oreMithril
        oreFluidCrudeOilShale
        oreFluidRedstone
        oreQuartz
        oreUranium
     >

    # Get additional attack power if a mob's max HP is over 20
    B:"Sword skill damage scaling"=false
}
`

**Comment by Helimyne** [comment]
after
`# Configuration file

blacklist {
    # Crops that won't be affected by farming growth skill, uses internal block name. No sync to client required. [default: []]
    S:"Crops for farming" <
        
     >
}


debug {
    # Forces Level Up! to restore external json files to default
    B:"Reset json files"=false
}


digloot {
    # Chances that a common loot drop will appear [range: 0 ~ 100, default: 85]
    I:"Common Digging Loot Chance"=85

    # Chances that a rare loot drop will appear [range: 0 ~ 100, default: 1]
    I:"Rare Digging Loot Chance"=1

    # Chances that an uncommon loot drop will appear [range: 0 ~ 100, default: 15]
    I:"Uncommon Digging Loot Chance"=15
}


general {
    # Always drop ore chunks on ore harvest
    B:"Always drop ore chunks"=false

    # Use ore chunks for ore doubling
    B:"Break ores into chunks"=true

    # All ores can be doubled, even if they don't have a chunk.
    B:"Duplicate any ore"=true

    # Colors for the end ore item
    I:"End Ore colors" <
        094747
        343434
     >

    # XP credit level for Mining Specialization
    I:"End Ore experience" <
        4
        2
     >

    # End ores that double from mining efficiency
    S:"End Ores to double" <
        oreFluidEnder
        oreFluidCrudeOilSand
     >

    # Does the furnace eject doubled items?
    B:"Furnace ejects bonus items"=false

    # Colors for the nether ore item
    I:"Nether Ore colors" <
        15064789
     >

    # XP credit level for Mining Specialization
    I:"Nether Ore experience" <
        2
        5
        4
        2
     >

    # Nether ores that double from mining efficiency
    S:"Nether Ores to double" <
        oreQuartz
        oreCobalt
        oreArdite
        oreFluidGlowstone
     >

    # Does the player lose all levels on death?
    B:"Reset class on death"=true

    # Colors for the surface ore item
    I:"Surface Ore colors" <
        3421236
        12360064
        16576075
        6155509
        1564002
        16711680
        1653940
        16739601
        9416910
     >

    # XP credit levels for Mining Specialization
    I:"Surface Ore experience" <
        1
        1
        2
        4
        4
        2
        2
        1
        1
        1
        1
        1
        2
        4
        3
        3
        1
        2
        2
        3
     >

    # Ores that double from mining efficiency
    S:"Surface Ores to double" <
        oreCoal
        oreIron
        oreGold
        oreDiamond
        oreEmerald
        oreRedstone
        oreLapis
        oreCopper
        oreTin
        oreSilver
        oreLead
        oreAluminum
        oreNickel
        orePlatinum
        oreIridium
        oreMithril
        oreFluidCrudeOilShale
        oreFluidRedstone
        oreQuartz
        oreUranium
     >

    # Get additional attack power if a mob's max HP is over 20
    B:"Sword skill damage scaling"=false
}


`

**Comment by BeetoGuy** [comment]
Oh, okay, I see what's going on, because you're using hexadecimal values, Minecraft is ignoring it. You have to convert the value to decimal first. You can do that [here.](http://www.binaryhexconverter.com/hex-to-decimal-converter)

**Comment by Helimyne** [comment]
hmmm, converted them all and the config for the stone based ores still trimmed the color list to default

**Comment by BeetoGuy** [comment]
It shouldn't be trimming it. I'm probably going to revert to the "Do Not Place" tag method in the next update, just to scale back on the level of config complication.

**Comment by Helimyne** [comment]
no, i think for now we'll use resource loader to manually assign textures, it doesn't delete the chunk or xp value after all
